lib/vagrant-proxyconf/action/base.rb

```diff
@@ -16,6 +16,8 @@ def call(env)
 
           if disabled? || !config.enabled?
```

Comment:
So this `disabled?` has nothing to do with `config.proxy.enabled` I guess. Bad overloading of the term, and even negative one which is almost always bad. Sorry. =)

Reply:
I think `disabled?` was a thing for each "module" (apt, git, ...) from early on. `config.proxy.enabled` was only introduced later, when someone needed a way to disable the whole plugin altogether depending on the provider. I should have realised the confusion already back then.

Reply:
It's all good.  It was easy enough to figure out after dropping into the debugger a few times to realize this is the main entry point to each action plugin.  I do wonder if we should remove the `!config.enabled?` conditional since it always seems to evaluate to `true` and `disabled?` seems to handle the globally disabling the proxy or just disabling the specific service.